addons/mail/models/mail_ice_server.py

```diff
@@ -10,7 +10,8 @@
 
 class MailIceServer(models.Model):
```

Comment:
in `_get_ice_servers`, don't forget to remove this:
```py
if self.env['ir.config_parameter'].sudo().get_param('mail.use_twilio_rtc_servers'):
```
since the check is not done in `get_twilio_credentials`.